.github/workflows/beam_PostCommit_Java_IO_Performance_Tests.yml

```diff
@@ -20,8 +20,6 @@ on:
     tags: ['v*']
     branches: ['master', 'release-*']
     paths: ['it/google-cloud-platform/**','.github/workflows/beam_PostCommit_Java_IO_Performance_Tests.yml']
-  issue_comment:
```

Comment:
these postcommit tests do not have jenkins counterpart, see https://github.com/apache/beam/issues/28426#issuecomment-1728293446

Reply:
Thanks for sharing the link, reverted changes back